hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleAccountState.java

```diff
@@ -69,7 +69,8 @@
 	static final int RELEASE_0180_VERSION = 7;
 	static final int RELEASE_0210_VERSION = 8;
 	static final int RELEASE_0220_VERSION = 9;
-	static final int RELEASE_0250_VERSION = 10;
+	static final int RELEASE_0230_VERSION = 10;
+	static final int RELEASE_0250_VERSION = 11;
```

Comment:
0240 is currently assigned version 11. Is it okay that we can't distinguish between 0240 and 0250 ?

Reply:
Unless we automate the process, it would be time-consuming and error-prone to bump the version number for every `SelfSerializable` type in ever release. 

So without such automation, our policy is to increment the version of a `SelfSerializable` type only in a release where its serde changes.